frontend/packages/ux-editor/src/containers/DesignView/FormTree/renderItemList.tsx

```diff
@@ -17,3 +18,19 @@ export const renderItemList = (
     </>
   ) : null;
 };
+
+export const renderItemListWithAddItemButton = (
+  layout: IInternalLayout,
+  duplicateComponents: string[],
+  parentId: string,
+) => {
+  const childIds = getChildIds(layout, parentId);
+  return (
+    <>
+      {childIds.map((id) => (
+        <FormItem duplicateComponents={duplicateComponents} layout={layout} id={id} key={id} />
+      ))}
```

Comment:
It looks like we can reuse `renderItemList` here
```suggestion
      {renderItemList(layout, duplicateComponents, parentId)}
```